Bottom: 879ab16b2866ccb99eb12bbbd72c20e63a0cecb7
Top:    9af18789fc4a504550058f002e75fbf7858e80e2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-19 18:17:34 +0100

Refresh of buffer-c-update-polygon-r-tree

---

diff --git a/src/buffer.c b/src/buffer.c
index 553c55f..2e63af5 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -201,9 +201,9 @@ AddPolygonToBuffer (LayerTypePtr Layer, PolygonTypePtr Polygon)
    * other objects, CreateNewPolygon does not do this as it creates a
    * skeleton polygon object, which won't have correct bounds.
    */
-  if (!Layer->polygon_tree)
-    Layer->polygon_tree = r_create_tree (NULL, 0, 0);
-  r_insert_entry (Layer->polygon_tree, (BoxTypePtr) polygon, 0);
+  if (!layer->polygon_tree)
+    layer->polygon_tree = r_create_tree (NULL, 0, 0);
+  r_insert_entry (layer->polygon_tree, (BoxType *)polygon, 0);
 
   CLEAR_FLAG (FOUNDFLAG | ExtraFlag, polygon);
   return (polygon);
